### ISSUE TITLE
'No such interface supported' after minor Windows update

### ISSUE THREAD

**Issue by MartinNiederl** [mrob95/pyvda#22]
After the latest minor Windows update I performed, pyvda suddenly fails directly on import (`from pyvda import VirtualDesktop`).

Platform version: 10.0.22621

Error log:

```
Connected to pydev debugger (build 232.9559.58)
Traceback (most recent call last):
  File "<frozen importlib._bootstrap>", line 1078, in _handle_fromlist
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "D:\dev\git\...\...\tray.py", line 10, in <module>
    from ....... import AutoSwitchLoop
  File "D:\dev\git\...\...\....py", line 7, in <module>
    from pyvda import VirtualDesktop
  File "D:\dev\git\...\.venv\lib\site-packages\pyvda\__init__.py", line 51, in <module>
    from .pyvda import (
  File "D:\dev\git\...\.venv\lib\site-packages\pyvda\pyvda.py", line 24, in <module>
    managers = Managers()
  File "D:\dev\git\...\.venv\lib\site-packages\pyvda\utils.py", line 55, in __init__
    self.manager_internal = get_vd_manager_internal()
  File "D:\dev\git\...\.venv\lib\site-packages\pyvda\utils.py", line 41, in get_vd_manager_internal
    return _get_object(IVirtualDesktopManagerInternal, CLSID_VirtualDesktopManagerInternal)
  File "D:\dev\git\...\.venv\lib\site-packages\pyvda\utils.py", line 30, in _get_object
    pServiceProvider.QueryService(
_ctypes.COMError: (-2147467262, 'No such interface supported', (None, None, None, 0, None))
```

**Comment by mrob95** [comment]
I've merged a fix for this (thanks to @mzomparelli) and released version `0.2.9`. I don't have a win11 PC to test this so let me know how it goes.

**Comment by MartinNiederl** [comment]
Thanks, now it works again as before!
